### PR TITLE
docs: translate the docs in English

### LINKED ISSUE #546: Translate the docs in English

### PR DESCRIPTION
**Pull request by kiwiwong**
## Description

Translate the docs in English

Fixes #546

### CODE REVIEW

**Comment by wewoor** [comment]
@SQLGate We are translating the Docs in this iteration #542, would you want to help us to review~

File: website/docs/guides/extend-builtin-ui.md [hunk 1]
```diff
@@ -144,32 +144,31 @@ molecule.search.onReplaceAll((tabId: UniqueId, groupId: UniqueId) => {
...
-**Output** 和 **Problems** 同为 **Panel** 的内置 UI 组件，目前我们并没有为 **Output** 面板提供独立的**服务（Service）**对象, 仍然需要借助 `molecule.panel` 服务来进行操作。
+**Output** and **Problems** are both **Panel**'s built-in UI components. At present, we do not provide a independent **Service** object for the **Output** panel, and we still need to use the `molecule.panel` service to operate.
 :::
 
-## [问题面板（Problems）](../api/interfaces/molecule.IProblemsService)
+## [Problems](../api/interfaces/molecule.IProblemsService)
 
-[Problems](../api/interfaces/molecule.IProblemsService) 可以用来展示工作区中的一些**问题**，例如**语法错误**、**参数问题**等。
-和 [Output](#输出面板output) 一样，同为 **Panel** 的内置部件。但不同的是，我们为 **Problems** 提供了独立的服务的对象，所有针对 Problems 操作的 API 是这样：
+[Problems](../api/interfaces/molecule.IProblemsService) can be used to show some **problems** in the workspace, such as **syntax errors**, **parameter problems**, etc. Like [Output](#output), it is also a built-in component of **Panel**. But the difference is that we provide an independent service object for **Problems**, and all APIs for Problems operations are like this:
```
Comment (mortalYoung) [inline website/docs/guides/extend-builtin-ui.md:171]: 这里需要和原本的 markdown 结构保持统一

File: website/docs/advanced/customize-workbench.md [hunk 2]
```diff
@@ -3,27 +3,27 @@ title: Custom Workbench
...
+## Custom Workbench Example
 
 ![Custom Workbench](/img/advanced/custom-workbench.png)
 
-Molecule 默认的是 **VSCode 布局**的 Workbench。在上图示例中，我们将 **MenuBar** 水平置于了**顶部**的位置，在编辑器的右侧，我们又自定义了一个**右边栏（RightSideBar)**。
+Molecule's Workbench defaults to **VSCode layout**. In the example above, we placed the **MenuBar** horizontally **at the top** position, and on the right side of the editor, we have customized a **RightSideBar**.
 
 :::info
-目前 **MenuBar 置顶布局**功能已内置，我们会在未来的版本中，将**右边栏（RightSideBar）**也作为 Molecule 的**内置**功能。
+At present, **MenuBar** has a built-in **top layout** function. We will use the **right sidebar (RightSideBar)** as a **built-in** function of Molecule in future versions.
```
Comment (mortalYoung) [inline website/docs/advanced/customize-workbench.md:21]:
```suggestion
At present, the top fixed layout for the **MenuBar** is integrated in Molecule. Besides, we will also integrate the **right sidebar (RightSideBar)** for Molecule in the future.
```

File: website/docs/advanced/customize-workbench.md [hunk 3]
```diff
@@ -3,27 +3,27 @@ title: Custom Workbench
 sidebar_label: Custom Workbench
 ---
 
-Molecule 默认的 **Workbench** UI 是一个 **VSCode** 的克隆版本。但是我们在实际的开发场景中，往往不能满足我们的需求。
+Molecule's default **Workbench** UI is a clone of **VSCode**. But in actual development scenarios, this often fails to meet our needs.
 
-除了内置的一些原子 [Components](../api/namespaces/molecule.component) 以外，Molecule 同时提供了基本的 **Workbench、SideBar、Editor、ActivityBar、MenuBar、Panel、StatusBar** 等核心[**UI 部件**](../guides/extend-workbench)，以便开发者根据自己的需求**重新组装**自己的 **Workbench**。
+In addition to some built-in atomic [Components](../api/namespaces/molecule.component), Molecule also provides basic **Workbench,SideBar, Editor, ActivityBar, MenuBar, Panel, StatusBar** and other core [**UI components**](../guides/extend-workbench), so that developers can **reassemble** their own **Workbench** according to their needs.
```
Comment (mortalYoung) [inline website/docs/advanced/customize-workbench.md:8]: 突然发现是不是都少了 mardown 的 list 结构 0.0

File: website/docs/advanced/customize-workbench.md [hunk 4]
```diff
@@ -3,27 +3,27 @@ title: Custom Workbench
...
 :::info
-目前 **MenuBar 置顶布局**功能已内置，我们会在未来的版本中，将**右边栏（RightSideBar）**也作为 Molecule 的**内置**功能。
+At present, **MenuBar** has a built-in **top layout** function. We will use the **right sidebar (RightSideBar)** as a **built-in** function of Molecule in future versions.
 :::
 
-### 重组 Workbench
+### Restructuring Workbench
 
-首先我们打开 Molecule [源码](https://github.com/DTStack/molecule)仓库，找到 `src/workbench` 目录，拷贝 `workbench.tsx` 文件到项目的 `views` 或其他目录下，将其重命名为 `myWorkbench.tsx` 文件：
+First, we open the [source](https://github.com/DTStack/molecule) code repository of Molecule, find the `src/workbench` directory, then copy the `workbench.tsx` file to the `views` or other directories of the project, and rename it to `myWorkbench.tsx`:
```
Comment (mortalYoung) [inline website/docs/advanced/customize-workbench.md:26]:
```suggestion
First, we open the [source code](https://github.com/DTStack/molecule) of Molecule, and find the `src/workbench` directory, then copy the `workbench.tsx` file to the `views` or other directories of your project, and rename it to `myWorkbench.tsx`:
```

**Review by mortalYoung — CHANGES_REQUESTED** [review]

**Review by SQLGate — COMMENTED** [review]
I wonder if the PR was requested after translating part of the document rather than translating the entire document.

**Review by SQLGate — APPROVED** [review]

File: website/docs/guides/extend-builtin-ui.md [hunk 5]
```diff
@@ -126,11 +125,12 @@ molecule.editorTree.onSelect((tabId: UniqueId, groupId: UniqueId) => {
 });
 ```
 
-## [搜索面板（Search)](../api/interfaces/molecule.ISearchService#setresult)
+## [Search](../api/interfaces/molecule.ISearchService#setresult)
 
-[Search](../api/interfaces/molecule.ISearchService#setresult) 是一个内置的**搜索面板**，它包含一些常见的**搜索**、**替换** 等 UI 功能。Molecule 内置的搜索模块只是基础的 UI 模块，具体的**搜索**，或者**替换**功能，需要开发者通过一些 API 来完成：
+[Search](../api/interfaces/molecule.ISearchService#setresult) is a built-in **search panel**, which contains some common **search**, **replace** and other UI functions. The built-in search module of Molecule is just a basic UI module. The specific **search** or **replacement** function needs to be completed by the developer through some APIs:
 
 监听**搜索输入控件**的输入内容：
```
Comment (mortalYoung) [inline website/docs/guides/extend-builtin-ui.md:132]: 这里忘记删了

File: website/docs/guides/extend-color-theme.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ Receiving objects: 100% (4493/4493), 19.68 MiB | 358.00 KiB/s, done.
 Resolving deltas: 100% (2830/2830), done.
 ```
 
-下载完成后，我们可以看到：
+After the download is complete, we can see:
```
Comment (mortalYoung) [inline website/docs/guides/extend-color-theme.md:32]:
```suggestion
After the download is completed, we can see:
```

File: website/docs/guides/extend-color-theme.md [hunk 7]
```diff
@@ -162,9 +162,9 @@ $ tree -L 2
...
 ```
 
--   `label` 属性的意思是当该主题被 Molecule 加载后，在颜色主题面板中显示的名称 `My Theme`。
--   `uiTheme` 属性的意思是该主题是暗黑主题还是亮色主题，抑或是高对比度主题.
--   `path` 属性的意思是该主题的详细主题配置所在的路径.
+-   The `label` attribute means the name `My Theme` displayed in the color theme panel after the theme is loaded by Molecule.
+-   The `uiTheme` attribute means whether the theme is a dark theme, a light theme, or a high-contrast theme.
+-   The `path` attribute means the path where the detailed theme configuration of the theme is located.
 
 然后，我们再创建一个 `themes/MyTheme.json` 文件，具体内容如下：
```
Comment (mortalYoung) [inline website/docs/guides/extend-color-theme.md]: 这句话没翻吗 0.0

File: website/docs/guides/extension.md [hunk 8]
```diff
@@ -26,7 +26,7 @@ export const ExampleExt: IExtension = {
 };
 ```
 
-### class 对象
+### class object
```
Comment (mortalYoung) [inline website/docs/guides/extension.md:29]:
```suggestion
### Class object
```

File: website/docs/quick-start.md [hunk 9]
```diff
@@ -4,46 +4,46 @@ sidebar_label: Quick Start
 sidebar_position: 2
 ---
 
-## 前置要求
+## Prerequisites
 
-> -   **Node.js 12.13.0 +** 版本
-> -   **React.js 16.14.0 +** 版本
-> -   [Yarn](https://yarnpkg.com/en/) - 推荐使用 **Yarn** 作为包管理
+> -   node.js version: **Node.js 12.13.0 +**
```
Comment (mortalYoung) [inline website/docs/quick-start.md:9]:
```suggestion
> -   Node.js version: **Node.js 12.13.0 +**
```

**Review by mortalYoung — CHANGES_REQUESTED** [review]

**Review by wewoor — CHANGES_REQUESTED** [review]
开头大写：
image

缺空格
image

**Review by mortalYoung — APPROVED** [review]
+1

**Review by wewoor — APPROVED** [review]
+1

**Pull request merged**